### PR TITLE
resources: add "How to change the wallet restore height" user guide

### PR DESCRIPTION
**Pull request by rating89us**

### CODE REVIEW

**Review by erciccione — APPROVED** [review]

**Review by erciccione — APPROVED** [review]

File: _i18n/ar/resources/user-guides/change-restore-height.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
+{% include disclaimer.html translated="no" translationOutdated="no" %}
+
+## How to change wallet restore height
```
Comment (erciccione) [inline _i18n/ar/resources/user-guides/change-restore-height.md:3]: This title is not necessary and should be removed

**Review by erciccione — CHANGES_REQUESTED** [review]

File: _i18n/ar/resources/user-guides/change-restore-height.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+### What is wallet restore height?
```
Comment (erciccione) [inline _i18n/ar/resources/user-guides/change-restore-height.md:5]: Add a newline after headers

File: _i18n/ar/resources/user-guides/change-restore-height.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+The wallet restore height is a setting that tells the wallet software at which block height it should start scanning the blockchain looking for transactions. Ideally it should be the number of the block which included the first incoming transaction to your wallet.
+
+### What happens if I set a wrong restore height?
```
Comment (erciccione) [inline _i18n/ar/resources/user-guides/change-restore-height.md:8]: Add a newline after headers

**Review by erciccione — CHANGES_REQUESTED** [review]

File: _i18n/ar/resources/user-guides/change-restore-height.md (hunk 1 above)
Comment (rating89us) [reply _i18n/ar/resources/user-guides/change-restore-height.md:3]: All user guides start with the article title. Why this guide should be different?

Comment (erciccione) [reply _i18n/ar/resources/user-guides/change-restore-height.md:3]: The title should be added in https://github.com/monero-project/monero-site/pull/1768/files#diff-21793b642d7e11b7e13418b727e75d96c62b4cf8d0d47100cda9a80ddb3d6fc0. I see now that it's missing. See for example https://raw.githubusercontent.com/monero-project/monero-site/master/resources/user-guides/importing_blockchain.md

**Comment by rating89us** [comment]
Ready to be reviewed

File: _i18n/ar/resources/user-guides/change-restore-height.md [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+
+- [What is the wallet restore height?](#what-is-the-wallet-restore-height)
+- [Why Monero wallets have a restore height?](#why-monero-wallets-have-a-restore-height)
+- [What happens if I set a wrong restore height?](#what-happens-if-i-set-a-wrong-restore-height)
+- [How to change the wallet restore height on Monero GUI](#how-to-change-the-wallet-restore-height-on-monero-gui)
+- [My wallet is showing a wrong balance after changing the restore height](#my-wallet-balance-is-wrong-after-changing-the-restore-height)
+
+### What is the wallet restore height?
+
+The wallet restore height is a setting that tells the wallet software at which block height it should start scanning the @blockchain looking for @transactions, a process that is known as wallet synchronization. Ideally it should be the number of the block which included the first incoming transaction to your wallet.
```
Comment (erciccione) [inline _i18n/ar/resources/user-guides/change-restore-height.md:13]:
```suggestion
The wallet restore height is a setting that tells the wallet software which block it should start scanning the @blockchain from looking for @transactions, a process that is known as wallet synchronization. Ideally it should be the number of the block which included the first incoming transaction to your wallet.
```

I would avoid using "height" if we can. People are often confused by the term and this guide could be very useful in clarifying things.

**Review by erciccione — APPROVED** [review]
Just a minor suggestion but overall looks good and very informative. Thanks for this and the other works.

File: _i18n/ar/resources/user-guides/change-restore-height.md (hunk 4 above)
Comment (rating89us) [reply _i18n/ar/resources/user-guides/change-restore-height.md]: fixed

**Review by erciccione — APPROVED** [review]

**Comment by rating89us** [comment]
fixed merge conflict

**Review by erciccione — APPROVED** [review]

**Comment by rating89us** [comment]
fixed merge conflict again

**Pull request merged**